python/ray/serve/_private/utils.py

```diff
@@ -569,3 +577,77 @@ def extract_self_if_method_call(args: List[Any], func: Callable) -> Optional[obj
                 return args[0]
 
     return None
+
+
+class MetricsPusher:
```

Comment:
Barely no change, and move it outside the autoscaling_policy.py. and start the MetricsPusher to manage lifecycle with clear interface.